metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/converters/HiveConnectorInfoConverter.java

```diff
@@ -76,7 +83,8 @@ public DatabaseInfo toDatabaseInfo(final QualifiedName qualifiedName, final Data
     public Database fromDatabaseInfo(final DatabaseInfo databaseInfo) {
         final QualifiedName databaseName = databaseInfo.getName();
         final String name = (databaseName == null) ? "" : databaseName.getDatabaseName();
-        final String dbUri = Strings.isNullOrEmpty(databaseInfo.getUri()) ? "" : databaseInfo.getUri();
+        //this is a temp hack to resolve the uri = null issue
+        final String dbUri = Strings.isNullOrEmpty(databaseInfo.getUri()) ? "file://temp/" : databaseInfo.getUri();
```

Comment:
Uri is set by the metastore by default if the uri is null. So we should just pass whatever is set by the user.